### ISSUE TITLE
Events and Actions page: some entries in Element-specific events table look broken

### ISSUE THREAD

**Issue by morsssss** [ampproject/amp.dev#4165]
Here, we're referring to the **Element-specific events** table on the [Events and Actions page](https://amp.dev/documentation/guides-and-tutorials/learn/amp-actions-and-events/). And we're looking at things in the **Data** column.

Looking at this page today, I assumed it had broken at some point... but, looking at past commits, I wonder if it's just simply never been finished.

Here's a milder case:

![Screen Shot 2020-07-08 at 4 20 36 PM](https://user-images.githubusercontent.com/29419498/86966370-0f2a1680-c137-11ea-8a3f-df020feda4f5.png)

Some of the entries in the `data` column are surrounded by `<pre>`.

```html
<pre>event.min
event.max
event.value
event.valueAsNumber</pre>
```

 Others are in `<code>`, as I think they should be:

```html
<code>event.checked</code>
```

Cases like this are more worrisome:

![Screen Shot 2020-07-08 at 4 12 24 PM](https://user-images.githubusercontent.com/29419498/86966738-a8f1c380-c137-11ea-9583-d8d0d806fcb8.png)

It looks like a TODO that was never to-done...

**Comment by mnjit20** [comment]
@morsssss how does it looks now
![image](https://user-images.githubusercontent.com/6074872/87724300-7e4cde00-c7bb-11ea-81af-6c2c2265b7d2.png)

and input elements
![image](https://user-images.githubusercontent.com/6074872/87724344-991f5280-c7bb-11ea-8d71-ab0218cdcf13.png)

I am going to make a PR with these changes

**Comment by mnjit20** [comment]
https://github.com/ampproject/amp.dev/pull/4193

**Comment by morsssss** [comment]
Thanks for formatting it! :) But we still need to make more sense out of the content in the right column. It shouldn't just be faux comments followed by object properties in code font.

I'm actually not sure what the best format would be here. Wonder if @CrystalOnScript could make a suggestion?

**Comment by mnjit20** [comment]
Sure if you @CrystalOnScript  could suggest then i will do the changes and maybe raise a new PR.

thanks

**Comment by mnjit20** [comment]
#4165 closed as the PR is merged

**Comment by patrickkettner** [comment]
fixed by #4193
